### PR TITLE
feature/PEER-191: Email notifications

### PR DESCRIPTION
**Pull request by alexeishulga**

### CODE REVIEW

File: app/containers/ChangeEmail/actions.js [hunk 1]
```diff
@@ -107,3 +106,18 @@ export function changeEmailErr(changeEmailError) {
     changeEmailError,
   };
 }
+
+export function getEmailAddress(address) {
```
Comment (Lampardks) [inline app/containers/ChangeEmail/actions.js:110]: why get? It's set

File: app/containers/ChangeEmail/actions.js [hunk 2]
```diff
@@ -107,3 +106,18 @@ export function changeEmailErr(changeEmailError) {
...
+  return {
+    type: GET_EMAIL_ADDRESS,
+    address,
+  };
+}
+
+export function getEmailAddressSuccess(email, isSubscribed) {
```
Comment (Lampardks) [inline app/containers/ChangeEmail/actions.js:117]: why get? It's set

add action for failed

Comment (alexeishulga) [reply app/containers/ChangeEmail/actions.js:118]: done: https://github.com/peeranha/peeranha-web/pull/1278/commits/94afbcc0fd515b56fbd935c9a160f522e4fc9d68

File: app/containers/ChangeEmail/actions.js (hunk 1 above)
Comment (alexeishulga) [reply app/containers/ChangeEmail/actions.js:111]: because the function makes a get request and returns data about user's email from back

**Pull request closed without merging**